src/psyclone/dynamo0p3.py

```diff
@@ -707,7 +707,7 @@ def __init__(self, ast, name=None):
     def _validate(self, need_evaluator):
         ''' Check that the meta-data conforms to Dynamo 0.3 rules for a
         user-provided kernel or a built-in '''
-        from dynamo0p3_builtins import BUILTIN_MAP
+        from psyclone.dynamo0p3_builtins import BUILTIN_MAP
```

Comment:
why is adding psyclone.* required in this issue?

Reply:
It's something I should have done when I re-structured the repository. Now pylint complains if we
import psyclone packages using a relative path so I'm fixing it as I come across it.